cmake/modules/FindTAEF.cmake

```diff
@@ -55,6 +55,16 @@ else()
   message(ERROR "Unable to find TAEF binaries.")
 endif()
 
+set(TAEF_ARCH "x86")
```

Comment:
Defaults to x86 if other expressions fail?  I'm not sure that's desirable.

Reply:
Fixed.